src/ssd/ssd-titlebar.c

```diff
@@ -593,9 +598,9 @@ ssd_update_window_icon(struct ssd *ssd)
 	 * was considered, but these settings have distinct purposes
 	 * already and are zero by default.
 	 */
-	int hpad = theme->window_button_width / 10;
-	int icon_size = MIN(theme->window_button_width - 2 * hpad,
-		theme->window_button_height);
+	int icon_padding = theme->window_button_width / 10;
+	int icon_size = MIN(theme->window_button_width - 2 * icon_padding,
+		theme->window_button_height - 2 * icon_padding);
```

Comment:
Let me confirm before merging. @jlindgren90 Is there any reason for applying the padding around the icon horizontally, but not vertically? I think it's more straightforward to apply padding in both directions.

Reply:
Ah I see. I didn't think about the space between icons since I usually don't place buttons next to the window icon. 

For simplification, I'll keep the patch as it is.